kurlkinds/config/crds/v1beta1/cluster.kurl.sh_installers.yaml

```diff
@@ -330,13 +330,16 @@ spec:
                 properties:
                   hostPath:
                     type: string
+                  migrateFromRGW:
+                    type: boolean
                   namespace:
                     type: string
                   s3Override:
                     type: string
                   version:
                     type: string
                 required:
+                - migrateFromRGW
```

Comment:
should this really be required? I think you left out omitEmpty here: https://github.com/replicatedhq/kURL/pull/1867/files#diff-28c4c4c36269f130733810745e60144350618987ea7bdfed3fe68699749ad485R130

Reply:
right